assets/i18n/en.json

```diff
@@ -289,14 +290,20 @@
     "I18N_LEARNER_DASHBOARD_EMPTY_FEEDBACK_THREADS": "It looks like you don't have any active feedback threads. Your feedback helps us improve the quality of our lessons. Head over to the library to learn something exciting and submit your valuable feedback!",
     "I18N_LEARNER_DASHBOARD_EMPTY_INCOMPLETE_COLLECTIONS": "It looks like you have no partially-complete collections at the moment. Head over to the library to start an exciting new collection!",
     "I18N_LEARNER_DASHBOARD_EMPTY_INCOMPLETE_EXPLORATIONS": "It looks like you have no partially-complete explorations at the moment. Head over to the library to start an exciting new exploration!",
+    "I18N_LEARNER_DASHBOARD_EMPTY_SKILL_PROFICIENCY_AFTER_LINK": "to start an exciting new topic!",
+    "I18N_LEARNER_DASHBOARD_EMPTY_SKILL_PROFICIENCY_BEFORE_LINK": "It look like you have not started any topic yet. Head over to the&nbsp;",
+    "I18N_LEARNER_DASHBOARD_EMPTY_SKILL_PROFICIENCY_LINK": "<a href=\"/learn/math\" class=\"inline-links\">classroom</a>&nbsp;",
```

Comment:
Oh, no no. You need only translate 'classroom'. So the code in the HTML file would be:
`<a href="/learn/math" class="inline-links"><span translate="I18N_CLASSROOM...."></span></a>`

Reply:
Done as per discussion!